src/Game/Systems/SoundSystem.cpp

```diff
@@ -54,58 +42,47 @@ bool SoundSystem::OnPlayerSpawned(const Events::PlayerSpawned &e)
 bool SoundSystem::OnInputCommand(const Events::InputCommand & e)
 {
     if (e.Command == "Jump" && e.Value > 0) {
-        if (e.PlayerID == -1) { // local player
-            playerJumps();
-            return true;
+        if (e.PlayerID == -1) {
+            playerJumps(LocalPlayer);
+        } else {
```

Comment:
Only the server gets all the input commands, the clients will only get "PrimaryFire" and "Reload". In SoundSystem::playerJumps(EntityWrapper player) you only let clients play the sound, Dosen't this mean that the else at row 47 will never do anything in SoundSystem::OnInputCommand(const Events::InputCommand & e)?